test/spec/mapbox_logo.js

```diff
@@ -0,0 +1,83 @@
+describe('mapbox_logo', function() {
```

Comment:
To get this file to be picked up by the test runner, add `<script src="spec/mapbox_logo.js"></script>` to `test/index.html`.